src/Mailru/Provider.php

```diff
@@ -53,7 +53,7 @@ protected function getUserByToken($token)
     protected function mapUserToObject(array $user)
     {
         return (new User())->setRaw($user)->map([
-            'id'       => $user['email'],
+            'id'       => $user['id'],
```

Comment:
this would be a breaking change i guess?

Reply:
Likely. But it is anyway broken for now (

Email is not an identifier. And MailRu provides an identifier.